compiler/src/dotty/tools/dotc/ast/Trees.scala

```diff
@@ -221,7 +221,7 @@ object Trees {
       }
     }
 
-    override def hashCode(): Int = uniqueId // for debugging; was: System.identityHashCode(this)
+    override def hashCode(): Int = System.identityHashCode(this)
```

Comment:
I could not find which `hashCode` we are overriding. If it is indeed so we should just remove this override.

Reply:
We are overriding Object's hashcode, which is the same. But that way it is clear what we want. In fact, maybe we can emphasize this better by making `hasCode` and `equals` final.

Reply:
Ok